Corzbank.Services/EmailRegistrationService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Corzbank.Services.Interfaces;
+using MailKit.Net.Smtp;
+using MailKit.Security;
+using MimeKit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Corzbank.Services
+{
+    public class EmailRegistrationService: IEmailRegistrationService
+    {
+        public void SendEmail(string email, string subject, string message)
+        {
+            MimeMessage emailMessage = new MimeMessage();
+
+            emailMessage.From.Add(new MailboxAddress("Corzbank", "corzbank@gmail.com"));
```

Comment:
"Corzbank", "corzbank@gmail.com". Put this to appsettings

Reply:
resolved